docs/file.rst

```diff
@@ -116,7 +116,7 @@ In real life scenarios we should try to use `with` statement. It will take care
 
     >>> with open('setup.py') as fobj:
     ...     for line in fobj:
-    ...         print line,
+    ...         print (line,)
```

Comment:
Remove the command please and also the space before the `(`

Reply:
Are you talking about this line: `print (line,)`
please let me know